e2e/nomostest/ntopts/new.go

```diff
@@ -101,6 +101,14 @@ func RequireKind(t testing.NTB) Opt {
 	return func(opt *New) {}
 }
 
+// RequireHelmArtifactRegistry requires the --helm-provider flag to be set to artifact_registry
+func RequireHelmArtifactRegistry(t testing.NTB) Opt {
```

Comment:
Do we also need a `RequireOCIArtifictRegistry` option?

Reply:
Do we need a follow up PR to add RequireOCIArtifictRegistry to the tests that need it? Or are you saying there aren't any?

Reply:
There aren't any currently written in a way that need it.

Reply:
Sounds like we're missing some tests...

Reply:
Perhaps. There are certain tests for helm-sync which don't have an analog with oci-sync, for example helm-sync supports token auth while oci-sync does not.

There are also still a number of oci-sync tests which are written to pull from a static image whereas these helm-sync tests push a chart during the test. The registry provider type is only applicable when pushing artifacts during the test.

Reply:
Can you file some bugs for this tech debt to make the tests more consistent/robust?